pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -140,6 +142,7 @@ func (o *SyncOptions) BindFlags(cmd *cobra.Command) {
 			"Options are:\n"+strings.Join(kcpfeatures.KnownFeatures(), "\n")) // hide kube-only gates
 	cmd.Flags().DurationVar(&o.APIImportPollInterval, "api-import-poll-interval", o.APIImportPollInterval, "Polling interval for API import.")
 	cmd.Flags().DurationVar(&o.DownstreamNamespaceCleanDelay, "downstream-namespace-clean-delay", o.DownstreamNamespaceCleanDelay, "Time to wait before deleting a downstream namespaces.")
+	cmd.Flags().StringVar(&o.SyncTargetLabels, "labels", o.SyncTargetLabels, "A set of key=value pairs to be used as a labels for the sync target in kcp")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&o.SyncTargetLabels, "labels", o.SyncTargetLabels, "A set of key=value pairs to be used as labels on the SyncTarget created in kcp")
```

Reply:
Should we also mention the fact that they would be comma-separated ?

Reply:
Also why not use a `StringSliceVar` as for `apiexports` or `resources` arguments ?
This would take care of the comma-separated part of it, as the doc mentions

> 
> ```
> --ss="v1,v2" --ss="v3"
> will result in
> 
> []string{"v1", "v2", "v3"}
> ```

Reply:
good suggestion, done!
now you can do
```
kubectl kcp workload sync west --replicas=0 --syncer-image=syncer --output-file=west-syncer.yaml \
     --labels=region=east,foo=bar \
     --labels=bar=baz
```

Reply:
Nice